vscode/package.json

```
@@ -49,6 +49,11 @@
     "configuration": {
       "type": "object",
       "properties": {
+        "zeekLanguageServer.zeekPath": {
+          "type": "string",
+          "default": "",
+          "description": "Full path to the zeek executable. If unset we will attempt to look it up in PATH."
```

We just want the directory, not the full path.

```suggestion
          "description": "Directory containing Zeek executables. If unset we will attempt to find them in PATH."
```